### DISCUSSION TITLE
Version 3.6.2 available.

### DISCUSSION THREAD

**Discussion by janiversen** [pymodbus-dev/pymodbus#1907] [Announcements]
Version 3.6.0 had problems uploading to pypi, due to our documentation changes and change to use pyproject.toml
Version 3.6.1 missed updating the version number in documentation

Version 3.6.2 is the first of the 3.6 series, which we proudly announce !!

API_CHANGES.rst only contain one entry (framer can now be an enum), so for most cases this version can be plugged without app changes.

We are very pleased to welcome 9 new contributors....it goes to show this is not a one man project !

V3.6.0, V3.6.1 and V3.6.2 have around 66 commits, but nearly all of those are quality improvements of the code, some bug solving and a few new functionality.

Thanks to all who helped make this release happen, and a happy christmas to the part of the world who celebrate it.

We will be back next year with even more exciting releases:
- Async backbone for all clients (sync is merely an interface class)
- Tighter coupling to home assistant (modbus integration)
- Simpler API for clients.

Look forward to see comments/issues and most of all pull requests.
Jan I.

**Comment by MAKOMO** [comment]
Thank you for leading this important effort that well! Happy Chirstmas to you too.
